addon/components/mdc-textfield.js

```diff
@@ -123,12 +135,18 @@ export default Ember.Component.extend(MDCComponent, {
    * @type {Object}
    */
   CLASS_NAMES: cssClasses,
+  rippleOptions() {
+    return {
+      isSurfaceActive: () => getElementProperty(this, 'querySelector', () => [{[MATCHES]: () => false}])('input, textarea')[0][MATCHES](':active')
```

Comment:
Can we break out some of the anonymous functions to separate variables?  This line is a bit jam packed.